test/source-map/test-source-map-consumer.js

```diff
@@ -583,6 +583,34 @@ define(function (require, exports, module) {
     assert.equal(mappings.length, 0);
   };
 
+  exports['test allGeneratedPositionsFor for position with multiple mappings'] = function (assert, util) {
```

Comment:
This adds tests for multiple mappings for a single generated position, however there are no tests for passing a column into `allGeneratedPositionsFor`. Please add tests for that.

Also, how is the column parameter expected to interact with the fuzzy searching? Please add a test for this case as well.